docs/development/build-instructions-gn.md

```diff
@@ -57,7 +70,9 @@ $ gclient sync --with_branch_heads --with_tags
 ```sh
 $ cd src
 $ export CHROMIUM_BUILDTOOLS_PATH=`pwd`/buildtools
-$ gn gen out/Default --args='import("//electron/build/args/debug.gn")'
+# this next line is needed only if building with sccache
+$ export GN_EXTRA_ARGS="${GN_EXTRA_ARGS} cc_wrapper=\"${PWD}/electron/external_binaries/sccache\""
```

Comment:
Why the `${GN_EXTRA_ARGS}` here?

Reply:
To Handle the edge case of not clobbering a pre-existing `GN_EXTRA_ARGS` setting

A first-time user wouldn't have anything like that, but I was thinking more about the use case of a maintainer copy-pasting this into a terminal